opendistro-elasticsearch-index-management.release-notes.md

```diff
@@ -1,3 +1,8 @@
+## Version 1.6.0.0 (2020-03-17)
+
+### New Features
+* Adds support for Elasticsearch 7.6.1 [PR #164](https://github.com/opendistro-for-elasticsearch/index-management/pull/164)
```

Comment:
We should provide details about all the changes for development.

Reply:
Add some, think these are good and enough.